pubspec.yaml

```diff
@@ -98,6 +98,12 @@ flutter:
     - assets/images/onboarding/onboarding3.png
     - assets/images/onboarding/onboarding4.png
     - assets/images/onboarding/onboarding5.png
+    - assets/images/onboarding/leaves/pointing_left/medium_dark_leaf.png
```

Comment:
can we export these as SVGs instead of pngs.

Reply:
They are now SVGs

Reply:
?????


![image](https://user-images.githubusercontent.com/8041901/112862906-e1e21780-907b-11eb-946a-72b117dcde06.png)